src/datafactory/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+0.6.0
++++++
+* dependency module update
```

Comment:
Please describe which bug is fixed in this version

Reply:
Updated.